### DISCUSSION TITLE
Rename Boxes to Factories

### DISCUSSION THREAD

**Discussion by stephendolan** [luckyframework/lucky#1282] [Feature requests & Ideas] [answered]
Factories seem to be the canonical term at this moment in history for referencing tools to build up application data.

From a searchability perspective, it's much more likely that someone will look for "lucky factory library", and it'd be nice if our framework terminology reflected and accommodated that search.

Here's a link to more discussion in our Discord:
https://discord.com/channels/743896265057632256/743896265057632259/768507936716357682

**Comment by jwoertink** [comment]
I think I'm indifferent to it. Either name is fine for me... also... I'm not seeing any sparklyness... 🤔

**Reply by paulcsmith** [reply]
Sparkles don't work on discussions yet :(

**Comment by matthewmcgarvey** [comment]
I've gotten used to boxes in lucky but it was an odd name coming in and I'm definitely more familiar with "factories"

**Comment by paulcsmith** [comment, marked as answer]
I think there is no strong opinion in favor of keeping Box, and there are a lot of good reasons to go with Factory so I think we should go with Factory 👍 

* https://github.com/luckyframework/avram/issues/498
* https://github.com/luckyframework/lucky_cli/issues/563
* https://github.com/luckyframework/website/issues/452
